bones/testdata/golden/exemplar-double-no-minimal/exemplar.Exemplar.Sample.js

```diff
@@ -30,22 +30,16 @@ function Sample(input) {
         field: "",  // string
         repeat: [0],  // repeated int32
       },
-      aMap: [  // map<string, bool>
-        {
-          key: "",  // string
-          value: false,  // bool
-        }
-      ],
-      aDeepMap: [  // map<string, SampleMessage2>
-        {
-          key: "",  // string
-          value: {  // SampleMessage2
-            weirdFieldName1: "",  // string
-            aStringList: [""],  // repeated string
-            aMsgList: [{}],  // repeated SampleMessage1 (see example above)
-          },
-        }
-      ],
+      aMap: {  // map<int32, bool>
+        "0": false,
+      },
+      aDeepMap: {  // map<string, SampleMessage2>
+        "": {
```

Comment:
I was wondering if `"key"` what make it more obvious, but like @camh- has pointed out elsewhere: then this special cases string keys without any true special meaning (the empty string is valid).